org_status/org_status.py

```diff
@@ -84,6 +85,7 @@ def get_argument_parser():
     parser.add_argument('--verbose', '-v', action='store_true')
     parser.add_argument('--hosts-only', '-o', action='store_true')
     parser.add_argument('--skip-host-checks', action='store_true')
+    parser.add_argument('--import-repos', type=str)
```

Comment:
Should be a more appropriate name and also have a short option name. Maybe `--use-repo-list`, `-l`.